yarn.lock

```diff
@@ -23237,3 +23257,8 @@ zen-observable@^0.8.0, zen-observable@^0.8.14:
   version "0.8.15"
   resolved "https://registry.yarnpkg.com/zen-observable/-/zen-observable-0.8.15.tgz#96415c512d8e3ffd920afd3889604e30b9eaac15"
   integrity sha512-PQ2PC7R9rslx84ndNBZB/Dkv8V8fZEpk83RLgXtYd0fwUgEjseMn1Dgajh2x6S8QbZAFa9p2qVCEuYZNgve0dQ==
+
+zksync-web3@^0.8.1:
```

Comment:
@codenamejason why zksync-web3 is here?

Reply:
I wonder if related to updating this package `@nomiclabs/hardhat-etherscan": "^3.1.5` @damianmarti

Reply:
> I wonder if related to updating this package `@nomiclabs/hardhat-etherscan": "^3.1.5` @damianmarti

I think that zksync-web3 is not a dependency from @nomiclabs/hardhat-etherscan. Maybe you switch to another branch, run a "yarn install" and the yarn.lock gets updated and then you switch to this new branch with the file modified...

Reply:
@codenamejason I found where this zksync-web3 stuff come from. It's a dependency from hardhat-deploy. It was updated at https://github.com/scaffold-eth/scaffold-eth/commit/e4afffc7e3d6c2bb575240b4de830703e8925548 by @ZakGriffith but he forgot to commit the updated yarn.lock